### DISCUSSION TITLE
Calling All Freelance Developers!

### DISCUSSION THREAD

**Discussion by Erin-Cecele** [thegetty/quire#220] [Development]
While our goal is to make Quire as approachable as possible, we know that lack of in-house development support is often a major hurdle in smaller institutions determining if Quire is the right fit for them. In an effort to provide more support for our community, **we would like to put together a pool of freelance developers who can offer their services to Quire users in need.** 

If you are interested in joining this pool, please respond to this thread with the following information:

Name
Website
Previous Quire projects (if any)

Then email us at quire@getty.edu so we can get your full contact information.

![undraw_Developer_activity_re_39tg](https://user-images.githubusercontent.com/62075223/144508293-a8d1f77a-e6ee-4667-b2b4-07a1405bb801.png)

**Comment by chrisdaaz** [comment]
Thank you for starting this thread!

**Name:** Chris Diaz
**Website:** https://chrisdaaz.github.io/
**Previous Quire Projects:** [_Nuclear Compensation: Lessons from Fukushima_](https://nuclear-compensation.northwestern.pub/), an edited collection of academic essays.

**Reply by Erin-Cecele** [reply]
Thanks so much @chrisdaaz. Great to know you're available to help others in the community. We'll be in touch!

**Reply by chrisdaaz** [reply]
Just adding a 2022 example here: https://city-public-value-and-capitalism.northwestern.pub/
